src/wp-includes/blocks/navigation-submenu.php

```diff
@@ -14,66 +14,6 @@
  * @param  bool  $is_sub_menu Whether the block is a sub-menu.
  * @return array Colors CSS classes and inline styles.
  */
-function block_core_navigation_submenu_build_css_colors( $context, $attributes, $is_sub_menu = false ) {
```

Comment:
How would one deprecate functions that are in block PHP files?

Add them to `src/wp-includes/deprecated.php`?

I'm not sure we can move them from the block file (at least in Gutenberg) for folks running plugin + WP < 6.3

Reply:
> How would one deprecate functions that are in block PHP files?
> 
> Add them to `src/wp-includes/deprecated.php`?
> 
> I'm not sure we can move them from the block file (at least in Gutenberg) for folks running plugin + WP < 6.3

@ramonjd I think they'd need to be retained in GB and have a `_deprecated_function` call added alongside a `deprecated` annotation. Plugins have a habit of calling the most obscure functions so best to keep it to avoid fatals.

Reply:
Ah got it. Thanks for the advice @peterwilsoncc 

I can get a Gutenberg PR that does that, e.g., 

```php
... 
 * @deprecated 6.3.0 Use wp_apply_colors_support() instead.
 *
 * @param  array $attributes  Block attributes.
 * @param  bool  $is_sub_menu Whether the block is a sub-menu.
 * @return array Colors CSS classes and inline styles.
 */
function block_core_navigation_submenu_build_css_colors() {
	_deprecated_function( __FUNCTION__, '6.3.0', 'wp_apply_colors_support' );
...
```

Reply:
Oh actually, I created a deprecated entry for it:

https://github.com/WordPress/wordpress-develop/pull/4762